restli-server/src/main/java/com/linkedin/restli/internal/server/response/ResponseUtils.java

```diff
@@ -84,6 +90,20 @@ static Object translateCanonicalKeyToAlternativeKeyIfNeeded(Object canonicalKey,
     }
   }
 
+  public static void getAbsentFieldsDefaultValues(RecordDataSchema dataSchema, DataMap dataMap)
+  {
+    for (RecordDataSchema.Field field : dataSchema.getFields())
+    {
+      if (!dataMap.containsKey(field.getName()))
+      {
+        if (field.getDefault() != null)
```

Comment:
combine these 2 conditions in one statement

Reply:
Since I added the else section, it seems to me it is ok to keep it separate?